recipe/emailverification/api/emailverify.go

```diff
@@ -30,7 +30,7 @@ func EmailVerify(apiImplementation models.APIImplementation, options models.APIO
 			return err
 		}
 		token, ok := readBody["token"]
-		if !ok {
+		if !ok || token == nil {
```

Comment:
What is the type of `ok`? And why do you have `|| token == nil`? Makes me think that you are not sure of the types.

Reply:
ok is to heck if the token key exists. token is an interface so we check for nil.